src/extension/features/accounts/row-split-months/settings.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  name: 'RowSplitMonths',
+  type: 'checkbox',
+  default: false,
```

Comment:
suggestion: Can I add this feature as true by default? :)

Reply:
I think we need to leave this false -- visual features like this often confuse members :/

Reply:
It makes sense, thanks!